pvlib/tests/bifacial/test_utils.py

```diff
@@ -91,17 +93,23 @@ def test__vf_ground_sky_2d(test_system_fixed_tilt):
     assert np.isclose(vf, vfs_gnd_sky[0])
 
 
-@pytest.mark.parametrize("vectorize", [True, False])
-def test_vf_ground_sky_2d_integ(test_system_fixed_tilt, vectorize):
-    ts, pts, vfs_gnd_sky = test_system_fixed_tilt
-    # pass rotation here since max_rows=1 for the hand-solved case in
-    # the fixture test_system, which means the ground-to-sky view factor
-    # isn't summed over enough rows for symmetry to hold.
-    vf_integ = utils.vf_ground_sky_2d_integ(
-        ts['rotation'], ts['gcr'], ts['height'], ts['pitch'],
-        max_rows=1, npoints=3, vectorize=vectorize)
-    expected_vf_integ = trapezoid(vfs_gnd_sky, pts, axis=0)
-    assert np.isclose(vf_integ, expected_vf_integ, rtol=0.1)
+def test_vf_ground_sky_2d_integ():
+    # test against numerical integration with vf_ground_sky_2d
+    x = np.linspace(0, 1, num=1000)
+    kwargs = dict(gcr=0.4, pitch=5.0, height=1.5)
+    vf_x = utils.vf_ground_sky_2d(rotation=-60, x=x, **kwargs)
+    vf_expected = trapezoid(vf_x, x, axis=0)
+
+    vf_actual = utils.vf_ground_sky_2d_integ(surface_tilt=60, **kwargs)
+    assert np.isclose(vf_expected, vf_actual)
+
+
+def test_vf_ground_sky_2d_integ_deprecated():
```

Comment:
```suggestion
@fail_on_pvlib_version(...)
def test_vf_ground_sky_2d_integ_deprecated():
```
Let's make sure docs and code gets cleaned in the future? v0.12 or v0.13, whatever feels right. I would also add the future version to deprecate in the docs admonition.